src/main/java/org/dinosaur/foodbowl/domain/member/application/MemberService.java

```diff
@@ -37,6 +38,41 @@ public class MemberService {
     private final BookmarkRepository bookmarkRepository;
     private final BlameRepository blameRepository;
 
+    @Transactional(readOnly = true)
+    public MemberProfileResponse getMemberProfile(Long profileMemberId, Long memberId) {
+        Member profileMember = memberRepository.findById(profileMemberId)
+                .orElseThrow(() -> new FoodbowlException(MEMBER_NOT_FOUND));
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new FoodbowlException(MEMBER_NOT_FOUND));
+
+        boolean isSelfProfile = profileMember == member;
```

Comment:
```suggestion
        boolean isSelfProfile = Objects.equals(profileMember, member);
```

객체 타입을 비교할 때 Objects.equals를 사용하는 것은 어떨까요? ㅎㅎ
profileMember와 member가 모두 검증된 객체이기는 하지만, 객체 간의 동등성을 비교할 때 Objects의 equals를 사용하는 것을 권장하기도 하더라구요 !

Reply:
`eqauls` 내부적으로 동일한 동작을 수행하기는 수행하지만, 연산자가 아닌 메서드를 사용함으로 동등성 비교라는 메시지를 확실히 전달받을 수 있는 것 같아요 👍